scripts/build.sh

```diff
@@ -300,6 +300,10 @@ function GetLLVMVersion
       LLVM_VERSION=llvm-16
       return 0
     ;;
+    17)
```

Comment:
Probably should drop support for 15 and 16, especially if 17-related changes were made (e.g. dropping constant expression stuff).